ballerina-shell/modules/shell-cli/src/main/resources/command.header.txt

```diff
@@ -1,2 +1,3 @@
 Welcome to Ballerina Shell REPL.
+Ballerina version : Ballerina Swan Lake Beta 1
```

Comment:
This shouldn't be hard coded

Reply:
Removed in ee61c122de135136141783b29a433deeabc874c3